sample/src/androidMain/AndroidManifest.xml

```diff
@@ -1,7 +1,8 @@
 <manifest package="com.juul.krayon.sample"
     xmlns:android="http://schemas.android.com/apk/res/android">
     <application>
-        <activity android:name=".DemoActivity">
+        <activity android:name=".DemoActivity"
+            android:exported="true">
```

Comment:
Just curious, why is `exported="true"` needed?

Reply:
Target bump to Android 12 requires explicit `exported` set on any activity with an intent filter. `true` allows that activity to be launched by 3rd party apps (like launchers), `false` means the intent filter is for internal use only.